src/main/java/com/spring/amazon/service/ReviewDtoParser.java

```diff
@@ -0,0 +1,32 @@
+package com.spring.amazon.service;
+
+import com.opencsv.CSVParser;
+import com.spring.amazon.dto.ReviewDto;
+import java.io.IOException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewDtoParser implements CsvParser<ReviewDto> {
+
+    @Override
+    public ReviewDto parse(String line) {
+        ReviewDto reviewDto = new ReviewDto();
+        CSVParser csvParser = new CSVParser();
```

Comment:
since CSVParser is a local variable it will be created every time we call the parse method, can we declare it as a class variable?

Reply:
done